### PR TITLE
Add YAML linter

### LINKED ISSUE #188: Add linter for yaml files
We don't have any checks for yaml files.
We could use [yamllint](https://github.com/adrienverge/yamllint) for local checks and [this action](https://github.com/marketplace/actions/yaml-lint) for checks on CI.

### PR DESCRIPTION
**Pull request by anhappdev**
Closes #188

### CODE REVIEW

File: tools/formatter/configs/yamllint.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+  - '.yamllint'
+
+ignore: |
+  .github/workflows/cla.yml
```
Comment (d-uzlov) [inline tools/formatter/configs/yamllint.yml:7]: Why do we ignore this file?

Comment (anhappdev) [reply tools/formatter/configs/yamllint.yml:7]: The file `cla.yml` is used in multiple repos organization wide by the admin. If we enforce `yamllint` on this file, we need to reformat it. That makes it different from other `cla.yml`, which makes the maintenance of this file difficult for the admin.
The file `LICENSE.md` is also ignored from `markdownlint` for the same reason.

Comment (d-uzlov) [reply tools/formatter/configs/yamllint.yml:7]: Could you add this info in the file, please?

Comment (anhappdev) [reply tools/formatter/configs/yamllint.yml:10]: Done

**Review by d-uzlov — APPROVED** [review]

**Pull request merged**